redis/cluster.py

```diff
@@ -267,6 +267,7 @@ class AbstractRedisCluster:
                 "READONLY",
                 "CLUSTER INFO",
                 "CLUSTER MEET",
+                "CLUSTER MYSHARDID",
```

Comment:
Is this command implemented?

Reply:
Yes
https://github.com/redis/redis-py/blob/28cc65c18cc4fb37ef14497c963eb181dba8d25d/redis/commands/cluster.py#L638